src/issuingPurchase/issuingPurchase.php

```diff
@@ -86,7 +86,7 @@ class IssuingPurchase extends Resource
         - issuingTransactionIds [string]: ledger transaction ids linked to this Purchase
         - status [string]: current IssuingCard status. Options: "approved", "canceled", "denied", "confirmed", "voided"
         - description [string]:  IssuingPurchase description. ex: "Office Supplies"
-        - metadata [dictionary object]: dictionary object used to store additional information about the Transfer object. ex: [authorizationId => "OjZAqj"]
+        - metadata [dictionary object]: object used to store additional information about the IssuingPurchase object. ex: [authorizationId => "OjZAqj"]
```

Comment:
dictionary object used to store additional information about the IssuingPurchase object. ex: [authorizationId => "OjZAqj"]

Reply:
igual está em transfer